ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -175,8 +176,7 @@ private static void startRestServer(Map<String, Object> configs) throws Exceptio
         final int port = randomFreeLocalPort();
         serverAddress = "http://localhost:" + port;
         configs.put(RestConfig.LISTENERS_CONFIG, serverAddress);
-        restApplication = KsqlRestApplication.buildApplication(new KsqlRestConfig(configs),
-                                                               new DummyVersionCheckerAgent());
+        restApplication = KsqlRestApplication.buildApplication(new KsqlRestConfig(configs));
```

Comment:
question: is out tests now starting the rest api with a normal version checker and actually calling home now? Looks like it... I think we still need the ability to pass in a no-op version checker.

Reply:
Good point! I updated this so we can pass the version checker for tests and us noOp/Dummy version checkers there.